client/landing/gutenboarding/onboarding-block/question/index.tsx

```diff
@@ -26,22 +27,32 @@ const Question: FunctionComponent< Props > = ( {
 	isActive,
 	label,
 	onExpand,
-} ) => (
-	<div className={ classNames( 'onboarding-block__question', className, { selected: isActive } ) }>
-		<span>{ label }</span>
-		<div>
-			{ isActive ? (
-				children
-			) : (
-				<>
-					<button className="onboarding-block__question-answered" onClick={ onExpand }>
-						{ displayValue }
-					</button>
-					<span>.</span>
-				</>
-			) }
-		</div>
-	</div>
-);
+} ) => {
+	const springProps = useSpring( {
+		opacity: 1,
+		fontSize: isActive ? 40 : 28, // transition to a bigger font when the question is active
```

Comment:
Now I see this is in conflict with issue https://github.com/Automattic/wp-calypso/issues/38687 so it will be probably removed if still considered distracting even with current animation.

Reply:
Let's just roll that in here and supersede #38704. We can close #38687 with this as well.

```suggestion
```